frontend/coprs_frontend/coprs/logic/complex_logic.py

```diff
@@ -82,12 +82,13 @@ def get_transitive_runtime_dependencies(cls, copr):
         return list(internal_deps), list(external_deps), list(non_existing)
 
     @classmethod
-    def delete_copr(cls, copr, admin_action=False):
+    def delete_copr(cls, copr, admin_action=False, force=False):
         """
         Delete copr and all its builds.
 
         :param copr:
         :param admin_action: set to True to bypass permission check
+        :param force: delete even if permission check fails if set to True
```

Comment:
well it depends if we want to not check the `user.can_edit` in every case the `delete_expired_projects` is called - I don't know this for sure since it's used on multiple places. Is it intended behaviour for `delete_expired_projects` to completely skip the `user.can_edit` check as I do it in this PR?

Reply:
I think the `admin_action` was added exactly for this purpose; deleting in `delete_expired_projects`.